roles/zabbix_server/tasks/initialize-mysql.yml

```diff
@@ -27,6 +27,7 @@
 
 - name: "MySQL Database prep"
   when: zabbix_server_database_creation
+  become: "{{ zabbix_server_dbhost_run_install }}"
```

Comment:
Why elevate here?  Its not required.

Reply:
Actually it is if you're supplying `zabbix_server_mysql_login_unix_socket`, which you might do to avoid creating additional credentials for administering your mysql-server. (Similar to how we `become_user: postgres` to avoid creating credentials for postgresql. Atleast I think that's why we do it that way?) This seems to be the most effortless setup.

I have some work in progress testing the roles on vms with vagrant. With these changes, the only tunings i need to install zabbix-server-mysql are [these https://github.com/eb4x/community.zabbix/blob/vagrant/Vagrantfile#L82-L86](https://github.com/eb4x/community.zabbix/blob/vagrant/Vagrantfile#L82-L86). And the mysql installation is simply installing [mariadb-server](https://github.com/eb4x/community.zabbix/blob/vagrant/extensions/vagrant/database-mysql.yml)

I'm curious to know how others get this installed.